### PR TITLE
[new release] phylogenetics (0.3.0)

### PR DESCRIPTION
**Pull request by pveber**
Algorithms and datastructures for phylogenetics

- Project page: <a href="https://github.com/biocaml/phylogenetics/">https://github.com/biocaml/phylogenetics/</a>

##### CHANGES:

- better documentation (thanks to Corentin Moumard)
- implementation of Muse and Gaut 94 evolution model
- few bug fixes and API changes

### CODE REVIEW

File: packages/phylogenetics/phylogenetics.0.3.0/opam [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+bug-reports: "https://github.com/biocaml/phylogenetics/issues"
+depends: [
+  "alcotest" {with-test}
+  "angstrom-unix"
+  "binning"
+  "biotk" {>= "0.2.0"}
+  "core" {>= "v0.16.0"}
+  "dune" {>= "3.6"}
+  "gsl"
+  "lacaml" {>= "10.0.2"}
```
Comment (mseri) [inline packages/phylogenetics/phylogenetics.0.3.0/opam:17]:
```suggestion
  "yojson"
  "lacaml" {>= "10.0.2"}
```

File: packages/phylogenetics/phylogenetics.0.3.0/opam [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+bug-reports: "https://github.com/biocaml/phylogenetics/issues"
+depends: [
+  "alcotest" {with-test}
+  "angstrom-unix"
+  "binning"
+  "biotk" {>= "0.2.0"}
+  "core" {>= "v0.16.0"}
+  "dune" {>= "3.6"}
+  "gsl"
+  "yojson"
```
Comment (mseri) [inline packages/phylogenetics/phylogenetics.0.3.0/opam:17]:
```suggestion
  "yojson" {>= "1.6.0"}
```
For Yojson.Safe.t

**Comment by mseri** [comment]
Thanks

**Comment by mseri** [comment]
@pveber please upstream the fix to the opam file

**Comment by pveber** [comment]
Thanks a lot Marcello! The missing dep has been added (biocaml/phylogenetics@5e7a528)

**Pull request merged**